From ebcb2385c3f841127a3b03d17a9284557adb1348 Mon Sep 17 00:00:00 2001
From: Steve Beaver <sbeaver@netgate.com>
Date: Thu, 7 Apr 2016 10:33:33 -0400
Subject: [PATCH] Remove xslt header check

Signed-off-by: Steve Beaver <sbeaver@netgate.com>
---
 configure.in | 24 ++++++++++++------------
 1 file changed, 12 insertions(+), 12 deletions(-)

diff --git a/configure.in b/configure.in
index e26f108..2b5e90e 100644
--- a/configure.in
+++ b/configure.in
@@ -49,6 +49,8 @@ AC_SUBST(PACKAGE_VERSION)
 LIBTOOL_VERSION_INFO=`echo ${PACKAGE_VERSION} | sed -e 's/\./:/g' `
 AC_SUBST(LIBTOOL_VERSION_INFO)
 
+pushd ../git
+
 if test -n "`git branch | grep '\* *master'`"; then
 	RELEASE_TAG="_trunk"
 fi
@@ -69,6 +71,8 @@ USERMAIL=`git config --get user.email`
 AC_SUBST(USERNAME)
 AC_SUBST(USERMAIL)
 
+popd
+
 NC_WORKINGDIR_PATH="/var/lib/$PACKAGE_NAME/"
 SETBIT=0
 SETUSER=""
@@ -105,9 +109,8 @@ AC_ARG_WITH([libxml2],
 		if test "$XML2_CONFIG" = "no"; then
 			AC_MSG_ERROR([Missing development package of libxml2.])
 		fi
-		CFLAGS="`$withval/xml2-config --cflags` $CFLAGS"
-		LIBXML2_LIBS="`$withval/xml2-config --libs`"
-		LIBS="$LIBXML2_LIBS $LIBS"
+		CFLAGS=" $CFLAGS"
+		LIBS=" $LIBS"
 		WITH_LIBXML2="$withval"
 		CONFIGURE_PARAMS="--with-libxml2=$withval $CONFIGURE_PARAMS"
 		AC_SUBST(LIBXML2_LIBS)
@@ -121,8 +124,8 @@ AC_ARG_WITH([libxslt],
 		if test "$XSLT_CONFIG" != "no"; then
 			AC_MSG_ERROR([Missing xslt-config to get XSLT flags for compilation.])
 		fi
-		CFLAGS="`$withval/xslt-config --cflags` $CFLAGS"
-		LIBS="`$withval/xslt-config --libs` $LIBS"
+		CFLAGS=" $CFLAGS"
+		LIBS=" $LIBS"
 		WITH_LIBXSLT="$withval"
 		CONFIGURE_PARAMS="--with-libxslt=$withval $CONFIGURE_PARAMS"
 	]
@@ -455,9 +458,8 @@ if test -z "$WITH_LIBXML2" ; then
 	if test "$XML2_CONFIG" = "no"; then
 		AC_MSG_ERROR([Missing development package of libxml2.])
 	fi
-	CFLAGS="`xml2-config --cflags` $CFLAGS"
-	LIBXML2_LIBS="`xml2-config --libs`"
-	LIBS="$LIBXML2_LIBS $LIBS"
+	CFLAGS=" $CFLAGS"
+	LIBS=" $LIBS"
 	AC_SUBST(LIBXML2_LIBS)
 fi
 AC_CHECK_LIB([xml2], [xmlDOMWrapReconcileNamespaces],
@@ -470,12 +472,11 @@ if test "$validation" = "yes" -o "$yang" = "yes"; then
 	if test -z "$WITH_LIBXSLT" ; then
 		AC_CHECK_PROG([XSLT_CONFIG], [xslt-config], [yes], [no], [$WITH_LIBXSLT $PATH])
 		if test "$XSLT_CONFIG" != "no"; then
-			CFLAGS="`xslt-config --cflags` $CFLAGS"
-			LIBS="`xslt-config --libs` $LIBS"
+			CFLAGS=" $CFLAGS"
+			LIBS=" $LIBS"
 		fi
 	fi
 	AC_CHECK_LIB([xslt], [xsltApplyStylesheet], [], AC_MSG_ERROR([Missing libxslt library]))
-	AC_CHECK_HEADERS([libxslt/transform.h libxslt/xsltInternals.h], [], AC_MSG_ERROR([Missing libxslt headers]))
 fi
 
 if test "$libssh" = "yes"; then
@@ -532,7 +533,6 @@ fi
 
 if test "$url" = "yes"; then
 	### libcurl ###
-	PKG_CHECK_MODULES([CURL], [libcurl])
 	AC_CHECK_LIB([curl], [curl_global_init], [], AC_MSG_ERROR([Missing libcurl.]))
 	AC_CHECK_HEADERS([curl/curl.h], [], AC_MSG_ERROR([Missing libcurl headers.]))
 	BUILDREQS="$BUILDREQS libcurl-devel"
-- 
1.9.1

