From e6806f6b56c9b477645933388a4a37ed5261f0f1 Mon Sep 17 00:00:00 2001
From: Steve Beaver <sbeaver@netgate.com>
Date: Fri, 25 Mar 2016 15:07:25 -0400
Subject: [PATCH] patch

Signed-off-by: Steve Beaver <sbeaver@netgate.com>
---
 configure.in | 23 ++++++++++-------------
 1 file changed, 10 insertions(+), 13 deletions(-)

diff --git a/configure.in b/configure.in
index a53e37c..09693c0 100644
--- a/configure.in
+++ b/configure.in
@@ -48,7 +48,7 @@ PACKAGE_VERSION=`cut -f1 ./VERSION | tr -d '\n'`
 AC_SUBST(PACKAGE_VERSION)
 LIBTOOL_VERSION_INFO=`echo ${PACKAGE_VERSION} | sed -e 's/\./:/g' `
 AC_SUBST(LIBTOOL_VERSION_INFO)
-
+pushd ../git
 if test -n "`git branch | grep '\* *master'`"; then
 	RELEASE_TAG="_trunk"
 fi
@@ -68,7 +68,7 @@ USERNAME=`git config --get user.name`
 USERMAIL=`git config --get user.email`
 AC_SUBST(USERNAME)
 AC_SUBST(USERMAIL)
-
+popd
 NC_WORKINGDIR_PATH="/var/lib/$PACKAGE_NAME/"
 SETBIT=0
 SETUSER=""
@@ -105,9 +105,8 @@ AC_ARG_WITH([libxml2],
 		if test "$XML2_CONFIG" = "no"; then
 			AC_MSG_ERROR([Missing development package of libxml2.])
 		fi
-		CFLAGS="`$withval/xml2-config --cflags` $CFLAGS"
-		LIBXML2_LIBS="`$withval/xml2-config --libs`"
-		LIBS="$LIBXML2_LIBS $LIBS"
+		CFLAGS=" $CFLAGS"
+		LIBS="$LIBS"
 		WITH_LIBXML2="$withval"
 		CONFIGURE_PARAMS="--with-libxml2=$withval $CONFIGURE_PARAMS"
 		AC_SUBST(LIBXML2_LIBS)
@@ -121,8 +120,8 @@ AC_ARG_WITH([libxslt],
 		if test "$XSLT_CONFIG" = "no"; then
 			AC_MSG_ERROR([Missing development package of libxslt.])
 		fi
-		CFLAGS="`$withval/xslt-config --cflags` $CFLAGS"
-		LIBS="`$withval/xslt-config --libs` $LIBS"
+		CFLAGS="$CFLAGS"
+		LIBS="$LIBS"
 		WITH_LIBXSLT="$withval"
 		CONFIGURE_PARAMS="--with-libxslt=$withval $CONFIGURE_PARAMS"
 	]
@@ -436,9 +435,8 @@ if test -z "$WITH_LIBXML2" ; then
 	if test "$XML2_CONFIG" = "no"; then
 		AC_MSG_ERROR([Missing development package of libxml2.])
 	fi
-	CFLAGS="`xml2-config --cflags` $CFLAGS"
-	LIBXML2_LIBS="`xml2-config --libs`"
-	LIBS="$LIBXML2_LIBS $LIBS"
+	CFLAGS="$CFLAGS"
+	LIBS="$LIBS"
 	AC_SUBST(LIBXML2_LIBS)
 fi
 AC_CHECK_LIB([xml2], [xmlDOMWrapReconcileNamespaces], [HAVE_LIBXML2_RECONNS="#define HAVE_XMLDOMWRAPRECONCILENAMESPACE"], [HAVE_LIBXML2_RECONNS="#undef HAVE_XMLDOMWRAPRECONCILENAMESPACE"]
@@ -450,8 +448,8 @@ if test "$validation" = "yes" -a -z "$WITH_LIBXSLT" ; then
 	if test "$XSLT_CONFIG" = "no"; then
 		AC_MSG_ERROR([Missing development package of libxslt.])
 	fi
-	CFLAGS="`xslt-config --cflags` $CFLAGS"
-	LIBS="`xslt-config --libs` $LIBS"
+	CFLAGS="$CFLAGS"
+	LIBS="$LIBS"
 fi
 
 if test "$libssh" = "yes"; then
@@ -508,7 +506,6 @@ fi
 
 if test "$url" = "yes"; then
 	### libcurl ###
-	PKG_CHECK_MODULES([CURL], [libcurl])
 	AC_CHECK_LIB([curl], [curl_global_init], [], AC_MSG_ERROR([Missing libcurl.]))
 	AC_CHECK_HEADERS([curl/curl.h], [], AC_MSG_ERROR([Missing libcurl headers.]))
 	BUILDREQS="$BUILDREQS libcurl-devel"
-- 
1.9.1

