From 5c3602514a66bbe579e3557a97a8d5c337a6bca5 Mon Sep 17 00:00:00 2001
From: Steve Beaver <sbeaver@netgate.com>
Date: Tue, 29 Mar 2016 11:37:44 -0400
Subject: [PATCH] Fix models

Signed-off-by: Steve Beaver <sbeaver@netgate.com>
---
 Makefile.in | 22 +++++++++++-----------
 1 file changed, 11 insertions(+), 11 deletions(-)

diff --git a/Makefile.in b/Makefile.in
index b300f3c..c9b14af 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -72,7 +72,7 @@ NC_WORKINGDIR_PATH = $(DESTDIR)/@NC_WORKINGDIR_PATH@
 SETBIT = @SETBIT@
 SETUSER = @SETUSER@
 SETGROUP = @SETGROUP@
-IDGIT = "built from git $(shell git show --pretty=oneline | head -1 | cut -c -20)"
+IDGIT = "built from git $(shell pushd ../git;git show --pretty=oneline | head -1 | cut -c -20;popd)"
 IDNOGIT = "released as version $(VERSION)"
 LIBS = @LDFLAGS@ @LIBS@
 CFLAGS = -Wall @CFLAGS@
@@ -95,7 +95,7 @@ PYSRCS = python/netconf.c \
 	python/session.c
 endif
 
-INCLUDES= -I$(abs_srcdir)/src
+INCLUDES= -I$(abs_srcdir)/src -I$(abs_srcdir)/../build/src  -I$(abs_srcdir)/../../../libxml2/2.9.2-r0/package/usr/include/libxml2
 
 ifeq "$(shell pwd)" "$(shell $(readlink) -f $(srcdir))"
 OBJDIR = .obj
@@ -199,7 +199,7 @@ LNCDS_FILES = dev-tools/lncdatastore/main.c \
 
 XML_SRCS = models/ietf-netconf-acm-config.rng.in
 
-BUILT_RNGS = $(XML_SRCS:models/%.rng.in=models/%.rng)
+BUILT_RNGS = $(XML_SRCS:../git/models/%.rng.in=../git/models/%.rng)
 
 models/%.rng: models/%.rng.in
 	@sed -e "s/\@DATAROOTDIR\@/$(subst /,\/,$(datarootdir))/g" $< > $@
@@ -295,16 +295,16 @@ rpm: tarball $(TARBALLPYAPI)
 	fi
 
 .PHONY: install
-install: all install-devel install-doc $(INSTALLPYAPI)
+install: all install-devel $(INSTALLPYAPI)
 	$(INSTALL) -m 775 -d $(DESTDIR)/$(libdir)
 	$(LIBTOOL) --mode=install cp $(NAME).la $(DESTDIR)/$(libdir)
 	$(LIBTOOL) --mode=finish $(DESTDIR)/$(libdir)
-	@if test "`id -u`" = "0"; then $(LDCONFIG); fi
+
 	[ -d "$(NC_WORKINGDIR_PATH)" ] || \
 		mkdir -p $(NC_WORKINGDIR_PATH)
 	chmod 700 $(NC_WORKINGDIR_PATH)
-	$(INSTALL_DATA) models/ietf-netconf-acm*.rng models/ietf-netconf-acm-schematron.xsl $(NC_WORKINGDIR_PATH)/
-	$(INSTALL_DATA) models/yin2yang.xsl $(NC_WORKINGDIR_PATH)/
+	$(INSTALL_DATA) ../git/models/ietf-netconf-acm*.rng ../git/models/ietf-netconf-acm-schematron.xsl $(NC_WORKINGDIR_PATH)/
+	$(INSTALL_DATA) ../git/models/yin2yang.xsl $(NC_WORKINGDIR_PATH)/
 	@if test "$$(($(SETBIT) & 1))" = "1"; then \
 		if test -n "$(SETUSER)"; then \
 			chown $(SETUSER) $(NC_WORKINGDIR_PATH) || \
@@ -348,10 +348,10 @@ install-devel: $(LNCTOOL) $(NAME).la $(HDRS_PUBL_ROOT) $(HDRS_PUBL_SUBDIR) $(NAM
 	$(INSTALL_DATA) $(NAME).pc $(DESTDIR)/$(libdir)/pkgconfig/$(NAME).pc;
 	[ -d $(DESTDIR)/$(bindir) ] || \
 	   	(mkdir -p $(DESTDIR)/$(bindir); chmod 755 $(DESTDIR)/$(bindir)); \
-	$(INSTALL_PROGRAM) $(LNCTOOL).install $(DESTDIR)/$(bindir)/lnctool; \
-	$(INSTALL_DATA) dev-tools/lnctool/rnglib/* $(DESTDIR)/$(datadir)/$(NAME)/rnglib/; \
-	$(INSTALL_DATA) dev-tools/lnctool/xslt/* $(DESTDIR)/$(datadir)/$(NAME)/xslt/;
-	$(INSTALL_DATA) dev-tools/lnctool/generator/* $(DESTDIR)/$(datadir)/$(NAME)/templates/;
+	$(INSTALL_PROGRAM) ../git/$(LNCTOOL).install $(DESTDIR)/$(bindir)/lnctool; \
+	$(INSTALL_DATA) ../git/dev-tools/lnctool/rnglib/* $(DESTDIR)/$(datadir)/$(NAME)/rnglib/; \
+	$(INSTALL_DATA) ../git/dev-tools/lnctool/xslt/* $(DESTDIR)/$(datadir)/$(NAME)/xslt/;
+	$(INSTALL_DATA) ../git/dev-tools/lnctool/generator/* $(DESTDIR)/$(datadir)/$(NAME)/templates/;
 
 .PHONY: install-doc
 install-doc: doc
-- 
1.9.1

